test/spark/test_overtime.py

```diff
@@ -0,0 +1,70 @@
+import os
+import time
+
+import numpy as np
+import pyspark
+import pytest
+from sklearn.datasets import load_iris
+
+from flaml import AutoML
+from flaml.tune.spark.utils import check_spark
+
+try:
+    from test.spark.custom_mylearner import *
+except ImportError:
+    from custom_mylearner import *
+
+from flaml.tune.spark.mylearner import lazy_metric
+
+os.environ["FLAML_MAX_CONCURRENT"] = "10"
+
+spark = pyspark.sql.SparkSession.builder.appName("App4OvertimeTest").getOrCreate()
+spark_available, _ = check_spark()
+skip_spark = not spark_available
+
+pytestmark = pytest.mark.skipif(
+    skip_spark, reason="Spark is not installed. Skip all spark tests."
+)
+
+
+def test_overtime():
```

Comment:
Could you add more tests. For instances: test `MyLazyLGBM`, `use_spark=False`.

Reply:
No more test cases are necessary. Other tests that do not specify the 'force_cancel' argument can cover code that is not covered in the 'test_overtime'.

Reply:
test for `MyLazyLGBM` should be added.